node/connection_base.js

```diff
@@ -113,6 +114,13 @@ TChannelConnectionBase.prototype.onTimeoutCheck = function onTimeoutCheck() {
     }
     if (self.lastTimeoutTime) {
         self.timedOutEvent.emit(self);
+    } else if (!self.remoteName &&
+        self.timers.now() - self.startTime >= self.channel.initTimeout) {
+        self.timedOutEvent.emit(self, errors.TimeoutError({
+            start: self.startTime,
+            elapsed: self.timers.now() - self.startTime,
```

Comment:
call now only once and stash its return please... better even to factor out an `elapesd` quantity that the log and if agree about exactl